docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -124,18 +128,53 @@ PUT my-index-000001/_doc/1
 <1> The `my_integer` field is mapped as an `integer`.
 <2> The `my_string` field is mapped as a `text`, with a `keyword` <<multi-fields,multi field>>.
 
+[[match-mapping-runtime-fields]]
+===== Mapping runtime fields in a dynamic template
+You can also map runtime fields in a dynamic template. For example, the
+following request adds a dynamic template named `dayofweek` that maps all
+incoming `string` fields as `keyword` fields. Although the `runtime` definition
+is blank, new `string` fields will be mapped as `keyword` fields based on the
+<<dynamic-field-mapping-types,dynamic mapping rules>> that {es} uses for adding
+field types to the mapping. Any `string` that doesn't pass date detection or
+numeric detection is automatically mapped as a `keyword` when `dynamic` is set
```

Comment:
do we need to reinforce that it is mapped as a runtime keyword rather than the "ordinary" keyword?